Update the software list

Ebennetteng opened the pull request:
Added a new vendor, thank you for your time and consideration.

Ebennetteng committed:
updated vendor

justmurphy commented:
Hello @Ebennetteng.  We have updated the way we intake the data now, so you will want to make the addition this YAML file: https://github.com/cisagov/log4j-affected-db/blob/develop/data/cisagov_L.yml.
For any questions, please consult the contribution guidance: https://github.com/cisagov/log4j-affected-db/blob/develop/CONTRIBUTING.md
OR reach out here :)

Ebennetteng committed:
add

Ebennetteng committed:
Merge branch 'develop' into update

Ebennetteng commented:
> Hello @Ebennetteng. We have updated the way we intake the data now, so you will want to make the addition this YAML file: https://github.com/cisagov/log4j-affected-db/blob/develop/data/cisagov_L.yml. For any questions, please consult the contribution guidance: https://github.com/cisagov/log4j-affected-db/blob/develop/CONTRIBUTING.md OR reach out here :)

Hi there, I've added the info to that file instead and I've tried to delete my changes from last time, thank you

justmurphy committed:
remove entry to markdown

justmurphy committed:
Update Logit.io product name

justmurphy reviewed (approved):
Thank you for the update.
